video/out/opengl/context_drm_egl.c

```diff
@@ -119,7 +121,7 @@ static bool init_gbm(struct ra_ctx *ctx)
         p->gbm.device,
         p->kms->mode.hdisplay,
         p->kms->mode.vdisplay,
-        GBM_BO_FORMAT_XRGB8888,
+        GBM_FORMAT_XRGB8888,
```

Comment:
This should probably be in a separate commit, together with the change below that changes the framebuffer depth.

Reply:
OK i'll spit those changes into a separate commit.